lib/adapters/sequelize.js

```diff
@@ -41,24 +41,124 @@ function translateOptions(spell, options) {
   if (offset) spell.$offset(offset);
 }
 
+const setScopeToSpell = (scope) => (spell) => {
+  if (scope.where) {
+    spell.$where(scope.where);
+  }
+  if (scope.order) {
+    spell.$order(scope.order);
+  }
+  if (scope.limit) {
+    spell.$limit(scope.limit);
+  }
+};
+
+const mergeScope = (scopes) => {
+  let merged = {};
+  for (const scope of scopes) {
+    if (scope.where) {
+      merged.where = Object.assign({}, merged.where, scope.where);
+    }
+    if (scope.order) {
+      merged.order = Object.assign({}, merged.order, scope.order);
+    }
+    if (scope.limit) {
+      merged.limit = scope.limit;
+    }
+  }
+  return merged;
+};
+
+/**
+ * parse scope
+ * @param {Object} scope
+ * @param {Spell} spell
+ * @returns
```

Comment:
- return type not defined
- a few examples about `parseScope()` arguments would be better